twine/commands/check.py

```diff
@@ -88,6 +89,12 @@ def check(dists, output_stream=sys.stdout):
             )
             description_content_type = 'text/x-rst'
 
+        if description_content_type not in _supported_readme_types:
+            output_stream.write(
+                'warning; `long_description_content_type` invalid.\n'
```

Comment:
Depending on the team's thoughts I could make a change to make this an outright failure.

The problem is if this type is wrong pypi will outright reject the package with a 400 error. 

I know that technically this tool is checking if the readme will _render_ so perhaps the type is a separate concern?

Reply:
IN the back and forth on this PR I ended up just making this an error